### PR TITLE
Add `project list` command

### PR DESCRIPTION
**Pull request by oren-codefresh**

### CODE REVIEW

File: cmd/commands/project.go [hunk 1]
```diff
@@ -324,3 +327,112 @@ var getInstallationNamespace = func(repofs fs.FS) (string, error) {
...
+	log.G().Infof("cloning git repository: %s", opts.CloneOptions.URL)
+	_, opts.FS, err = opts.CloneOptions.Clone(ctx, opts.FS)
+	if err != nil {
+		return err
+	}
+	if !opts.FS.ExistsOrDie(store.Default.BootsrtrapDir) {
+		log.G().Fatalf("Bootstrap folder not found, please execute `repo bootstrap --installation-path %s` command", opts.FS.Root())
+	}
+
+	matches, err := billyUtils.Glob(opts.FS, "/projects/*.yaml")
```
Comment (noam-codefresh) [inline cmd/commands/project.go:398]: You can use `store.Default.ProjectsDir`, instead of hard-coding "projects"

File: cmd/commands/project.go [hunk 2]
```diff
@@ -324,3 +327,112 @@ var getInstallationNamespace = func(repofs fs.FS) (string, error) {
...
+			return err
+		}
+		fmt.Fprintf(w, "%s\t%s\t%s\n", proj.Name, proj.Namespace, proj.ClusterName)
+
+	}
+	w.Flush()
+	return nil
+}
+
+func getProjectInfoFromFile(fs fs.FS, name string) (*argocdv1alpha1.AppProject, error) {
```
Comment (roi-codefresh) [inline cmd/commands/project.go:417]: I think this should return 3 types `(*argocdv1alpha1.AppProject, *ApplicationSet, error)` even if we don't need the application set now

File: cmd/commands/project_test.go [hunk 3]
```diff
@@ -300,3 +304,64 @@ spec:
...
+			},
+			want: &argocdv1alpha1.AppProject{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      "prod",
+					Namespace: "ns",
+				},
+			},
+		},
+	}
+	for tNAME, tt := range tests {
```
Comment (roi-codefresh) [inline cmd/commands/project_test.go:351]: why capital `tNAME`?

**Review by roi-codefresh — APPROVED** [review]

File: cmd/commands/project.go (hunk 1 above)
Comment (roi-codefresh) [reply cmd/commands/project.go:398]: also use `fs.Join()`

File: cmd/commands/common.go [hunk 4]
```diff
@@ -68,8 +69,13 @@ var (
 		}
 
 		log.G().Debug("repository is ok")
+
 		return r, repofs, nil
 	}
+
+	glob = func(fs fs.FS, pattern string) ([]string, error) {
```
Comment (roi-codefresh) [inline cmd/commands/common.go:76]: can this be turned into `glob = billyUtil.Glob`?

**Review by roi-codefresh — APPROVED** [review]

**Pull request merged**